application/src/main/java/org/togetherjava/tjbot/commands/moderation/temp/RevocableModerationAction.java

```diff
@@ -12,21 +12,14 @@
  */
 interface RevocableModerationAction {
     /**
-     * Classification of a revocation failure.
+     * The name of the action, that will be logged in case of a failed revocation.
+     *
+     * <p>
+     * Naming convention examples: {@code "ban"}, {@code "mute"}, {@code "quarantine"}
+     * </p>
      */
-    @SuppressWarnings("PublicInnerClass")
-    enum FailureIdentification {
-        /**
-         * Acknowledges that the failure is known and has been handled. Hence, further error
-         * handling should not be continued.
-         */
-        KNOWN,
-        /**
-         * The failure is unknown and has not been handled. Hence, further error handling should be
-         * continued.
-         */
-        UNKNOWN
-    }
+    @NotNull
+    String actionName();
```

Comment:
You can use the constructor instead, that's shorter in almost all cases

Reply:
(its an interface though :D)

Reply:
My bad, alcohol is bad kids